From 00e6e985c4fa5fc7e8eabca077145aebce493c00 Mon Sep 17 00:00:00 2001
From: Jiri Moskovcak <jmoskovc@redhat.com>
Date: Thu, 18 Oct 2012 18:07:01 +0200
Subject: [PATCH 67/67] added relro to reportclient.so and _pyreport.so
 rhbz#812283

Conflicts:
	src/client-python/Makefile.am
	src/report-python/Makefile.am
---
 src/client-python/Makefile.am | 3 ++-
 src/report-python/Makefile.am | 1 +
 2 files changed, 3 insertions(+), 1 deletion(-)

diff --git a/src/client-python/Makefile.am b/src/client-python/Makefile.am
index 74486f2..f5f8528 100644
--- a/src/client-python/Makefile.am
+++ b/src/client-python/Makefile.am
@@ -19,7 +19,8 @@ _reportclient_la_CPPFLAGS = \
     $(GLIB_CFLAGS) \
     $(PYTHON_CFLAGS) \
     -D_GNU_SOURCE \
-    -Wall -Wwrite-strings -Werror
+    -Wall -Wwrite-strings -Werror \
+    -fPIE -Wl,-z,bind_now
 _reportclient_la_LDFLAGS = \
     -module \
     -avoid-version \
diff --git a/src/report-python/Makefile.am b/src/report-python/Makefile.am
index e12479f..61af4d2 100644
--- a/src/report-python/Makefile.am
+++ b/src/report-python/Makefile.am
@@ -24,6 +24,7 @@ _pyreport_la_CPPFLAGS = \
     $(PYTHON_CFLAGS) \
     -D_GNU_SOURCE \
-    -Wall -Wwrite-strings -Werror
+    -Wall -Wwrite-strings -Werror \
+    -fPIE -Wl,-z,bind_now
 _pyreport_la_LDFLAGS = \
     -module \
     -avoid-version \
-- 
1.7.12.1

